main.go

```diff
@@ -122,6 +126,18 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "ClusterScan")
 		os.Exit(1)
 	}
+
+	if err = (&zoracontrollers.SaasReconciler{
+		Client:     mgr.GetClient(),
+		Scheme:     mgr.GetScheme(),
+		HttpCli:    &http.Client{},
+		ID:         saasID,
+		ServerAddr: saasServerAddr,
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "Saas")
+		os.Exit(1)
+	}
```

Comment:
here we can have a condition to create this controller...
if `saasID` is empty, it means that the saas isn't being used and this controller does not need to be created..

Reply:
Done